scrapers_next/or/committee.py

```diff
@@ -0,0 +1,61 @@
+from spatula import XPath, URL, HtmlListPage, HtmlPage, CSS
+from openstates.models import ScrapeCommittee
+import re
+
+class CommitteeDetail(HtmlListPage):
+    source = URL("https://olis.oregonlegislature.gov/liz/2023R1/Committees/JCT/CommitteeMembers")
+    selector = CSS('tbody tr')
+
+    def process_item(self, item):
+        com = self.input
+        position, name = item.getchildren()
+        name = re.sub(r"Senator |Representative ", "", name.text_content().strip())
+        position = position.text_content().strip()
+        com.add_member(name, position)
+
+        return com
```

Comment:
While it seems intuitive to use `HtmlListPage` here, this is currently returning the committee object (and consequently writing an additional JSON file) for every item processed on the page using the `process_item()` method. That means we end up with as many committee objects as there are total members across all committees.

Instead, you can simply reformat what you've already done into a `HtmlPage` class like this:
```
class CommitteeDetail(HtmlPage):
    example_source = URL("https://olis.oregonlegislature.gov/liz/2023R1/Committees/JCT/CommitteeMembers")

    def process_page(self):
        com = self.input
        members = CSS('tbody tr').match(self.root)
        for member in members:
            position, name = member.getchildren()
            name = re.sub(r"Senator |Representative ", "", name.text_content().strip())
            position = position.text_content().strip()
            com.add_member(name, position)
        return com
```

That should serve this purpose better. While the list page scraper is very helpful -- and a huge reason we love spatula -- details pages (i.e. your final destination for scraping each committee) are typically best left to a normal page type class.